src/simput/CellEditor.js

```diff
@@ -145,91 +143,64 @@ export default class CellEditor extends React.Component {
     const materials =
       'materials not found' in this.props.ui.domain ? {} : this.props.ui.domain;
 
+    const materialOptions = Object.keys(materials).map((key) => (
+      <option key={key} value={materials[key].name}>
+        {materials[key].name}
+      </option>
+    ));
+
+    const columns = [
+      {
+        key: 'material',
+        dataKey: 'material',
+        label: 'Material',
+        render: (mat, item) => (
+          <select
+            onChange={(e) => this.onMaterialChange(item, e.target.value)}
+            value={mat}
+            className={style.material}
+          >
+            {materialOptions}
+          </select>
+        ),
+      },
+      {
+        key: 'radius',
+        dataKey: 'radius',
+        label: 'Radius',
+        render: (radius, item) => (
+          <input
+            className={style.radius}
+            type="number"
+            step="0.01"
+            max={this.state.pinPitch * 0.5}
```

Comment:
It is valid to enter a pin with a radius that goes beyond the cell, if the neighboring pins are small enough to allow it. I wouldn't specify a max.

Reply:
Right, I'll take care of that.